src/Services/PageService.cs

```diff
@@ -42,10 +53,24 @@ public PageService()
             }
         }
 
-        var experimentalFeaturesVM = App.Current.GetService<ExperimentalFeaturesViewModel>();
-        foreach (var experimentId in App.NavConfig.ExperimentIds ?? Array.Empty<string>())
+        var experimentationService = Application.Current.GetService<IExperimentationService>();

```

Comment:
Consider depending on the object from the constructor

Reply:
Fixed!